src/test/kotlin/no/nav/tilleggsstonader/sak/util/TestoppsettService.kt

```diff
@@ -96,10 +104,36 @@ class TestoppsettService(
         return fagsak.tilFagsakMedPerson(person.identer, eksternFagsakId)
     }
 
+    fun opprettRevurdering(
+        revurderFra: LocalDate,
+        forrigeBehandling: Behandling,
+        fagsak: Fagsak,
+        vedtakBeregningsresultatForrigeBehandling: BeregningsresultatTilsynBarn = vedtakBeregningsresultat,
+    ): Behandling {
+        oppdater(forrigeBehandling.copy(status = BehandlingStatus.FERDIGSTILT))
+        val forrigeVedtak = innvilgetVedtak(
+            behandlingId = forrigeBehandling.id,
+            beregningsresultat = vedtakBeregningsresultatForrigeBehandling,
+        )
+        repository.insert(forrigeVedtak)
```

Comment:
Jeg føler ikke at det er helt intuitivt at det skal lagres et innvilget vedtak her, men er sikkert greit 👀

Reply:
Hmm, ja det er kanskje litt rart? Kan legge funksjonaliteten for å lage vedtak i egen funksjon? [f493574](https://github.com/navikt/tilleggsstonader-sak/pull/483/commits/f4935744239737d02c08148e8c04af672df450b0)